backend/utils/converters.go

```diff
@@ -0,0 +1,21 @@
+package utils
+
+import (
+	"github.com/google/uuid"
+	"github.com/samber/lo"
+)
+
+// AsUuid parses string as Uuid, and returns it or an empty uuid.
+func AsUuid(s string) uuid.UUID {
+	uid, _ := uuid.Parse(s)
+	return uid
+}
```

Comment:
Should we have a warn log here, in case the UUID passed in is in fact invalid?

Reply:
I don't think it'll help us with anything, as it just tells us if theres an invalid UUID _somewhere_ in the system. 

Maybe as a comment specify that it's only used when it is a uuid? At the moment I know for a fact that the only ids sent through the function are uuids, but in the odd case where it isn't, I'm fine with them being treated as empty uuids as well.

Reply:
If you wrap the err with merry you could make it print the stack. That should give a clue/starting point.